docs/react/guides/dependent-queries.md

```diff
@@ -71,7 +71,7 @@ const { data: userIds } = useQuery({
 // Then get the users messages
 const usersMessages = useQueries({
   queries: users
```

Comment:
In the context of the code above, shouldn't this be modified too?

```suggestion
  queries: usersIds
```

Reply:
Yeah, I think so, makes sense. I missed that bc I was fixated at the `userIds`. This would work as far as I see.

```js
const usersMessages = useQueries({
  queries:
    userIds?.map(id => ({
      queryKey: ['messages', id],
      queryFn: () => getMessagesByUsers(id)
    })) ?? [] // if userIds is undefined, an empty array will be returned
});
```